app/common/lib/ledgerUtil.js

```diff
@@ -106,3 +127,90 @@ module.exports.walletStatus = (ledgerData) => {
   }
   return status
 }
+
+// TODO rename function
+const blockedP = (state, publisherKey) => {
```

Comment:
Super niceP having this logicP pulled out of ledger.js.  Makes me happyP!  Good jobP! 😆

Reply:
:smile: